index.js

```diff
@@ -23,6 +23,62 @@ function addNumbers(num1, num2) {
 // 👇 COMPLETE YOUR WORK BELOW 👇
 // 👇 COMPLETE YOUR WORK BELOW 👇
 
+/**
+ * ### Challenge `sayGoodbye`
+ * 
+ * @instructions
+ * This function should take an a name as an argument,
+ * and return a string that says 'Goodbye, {name}. Have a great day.'
+ * 
+ * For example, if we invoke `sayGoodbye`
+ * passing 'Andy' as the argument,
+ * the returned value should look like: 'Goodbye, Andy. Have a great day.'
+ * 
+ * Reminder: Comment in the test for this function in `tests/index.test.js`
+*/
+function sayGoodbye(/* code here */) {
+  /* code here */
+}
+
+/**
+ * ### Challenge `temperatureCtoF`
+ * 
+ * @instructions
+ * This function should take an a temperature in celsius as an argument,
+ * and return the temperature in fahrenheit, rounded to the nearest whole number. 
+ * 
+ * For example, if we invoke `temperatureCtoF`
+ * passing 24 as the argument,
+ * the returned value should be: 75
+ * 
+ * Hint 1: The formula for converting celsius to fahrenheit is t*9/5 + 32 where t is the temperature in celsius.
+ * Hint 2: There is a very easy way to round numbers in JS. Do a google search to find out how. 
+*/
+function temperatureCtoF(/* code here */) {
+  /* code here */
+}
+
+/**
+ * ### Challenge `temperatureInF`
```

Comment:
It'd be nice to point out we can use the previous function temperatureCtoF as a helper inside of this one